src/FilterBundle/DependencyInjection/BestItCommercetoolsFilterExtension.php

```diff
@@ -32,6 +32,9 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setAlias('best_it_commercetools_filter.request.client', $config['client_id']);
         $container->setAlias('best_it_commercetools_filter.generator.url', $config['url_generator_id']);
 
+        // Set parameters
+        $container->setParameter('best_it_commercetools_filter.cache_life_time', $config['cache_life_time']);
```

Comment:
Configuration component need this value too. at least with a default value. see https://github.com/bestit/commercetools-filter-bundle/blob/master/src/FilterBundle/DependencyInjection/Configuration.php